### PR TITLE
158421514 signup

### PR DESCRIPTION
**Pull request by xcixor**
#### Purpose of the PR
Creates the signup page
#### Tasks to be complete
* Create the signup form
#### Manual testing
* Navigate to the repo's main page via  [this link](https://github.com/xcixor/my-ride)
* Under the repo's name copy the clone's url
* Open a terminal and navigate to where you want the repo to be downloaded
* Type git clone 'the url you copied above' and press enter
* Once downloaoded navigate to the UI folder
* Open the index file with your favorite browser
* Click on Join us in the navbar
* To preview on mobile in chrome inspect the page and the browser's inspection tool will provide 
   previews for various screens
#### Relevant pivotal tracker stories 
#158421514 User should be able to signup 

##screenshots
signdesk
signmob

### CODE REVIEW

**Comment by ghost** [comment]
the code is okay and the design is good

**Comment by artorious** [comment]
Looks good. KeepItPushing.
Consider editing your PR description (especially testing instructions) and maybe make `#158421514 User should be able to signup` a link.

Who is the lady though? Satisfied customer?

File: UI/static/css/signup.css [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+.sign-up {
+    height: 600px;
+    grid-area: sign-up;
+    background-color: whitesmoke;
+    justify-self: center;
+    border-radius: 10px;
+    display: grid;
+    grid-template-areas: "profile-img" "sign-up-form";
+    /* padding: 10px;
```
Comment (JoshuaOndieki) [inline UI/static/css/signup.css]: Remove unused code

**Review by JoshuaOndieki — CHANGES_REQUESTED** [review]

File: UI/static/css/signup.css (hunk 1 above)
Comment (xcixor) [reply UI/static/css/signup.css]: Always forget this, let me fix that

**Pull request merged**